onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -79,6 +79,7 @@
   return true;
 }
 
+template <typename AType>
```

Comment:
>AType [](http://example.com/codeflow?start=19&length=5)

Do you need a template for whole class? The data type of A and scale can be retrieved from Tensor A and Tensor Scale

Reply:
removed the template.